parquet-hadoop/src/test/java/org/apache/parquet/filter2/recordlevel/PhoneBookWriter.java

```diff
@@ -340,12 +342,21 @@ public static void write(ParquetWriter.Builder<Group, ?> builder, List<User> use
     return users;
   }
 
-  public static List<User> readUsers(ParquetReader.Builder<Group> builder) throws IOException {
```

Comment:
Removing public method could cause incompatibility issue.

Reply:
fixed - not deleting it.